Common/src/main/java/com/techarium/techarium/multiblock/MultiblockStructure.java

```diff
@@ -31,25 +31,21 @@
  * <br>
  * Instances of this class are created via datapack.
  */
-public class MultiBlockStructure {
+public class MultiblockStructure {
```

Comment:
If we're going with uncapitalized b, you should rename usage of MultiBlock to use a lowercase b.